scripts/package_building/dh_builddeb

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/perl
```

Comment:
create a scripts/package_building/kernel_metapackage
and all the metapackage stuff goes in there